feat(changelog): load-balancers-added-http-redirect-and-unconditional 2023-04-11 (#1520)
scaleway/docs-content: changelog/april2023/2023-04-11-load-balancers-added-http-redirect-and-unconditional.mdx

---

> title: HTTP redirect and unconditional ACLs just landed

Reviewer: ```suggestion
title: HTTP redirect and unconditional ACLs are now available!
```

status: added
author:
    fullname: 'Join the #load-balancer channel on Slack.'
    url: 'https://scaleway-community.slack.com/archives/load-balancer'
date: 2023-04-11
category: network
product: load-balancers
---

Load Balancer now lets you redirect client requests to alternative URLs. You can use this feature to secure your service by redirecting all HTTP traffic to HTTPS, or to redirect users to a backup service when carrying out maintenance.

> HTTP redirects come as a new type of ACL configured within a frontend. You can fully customize the redirection URL, from the protocol scheme all the way to the query parameters, using powerful templated place holders to reuse parts of the original requests.

Reviewer: ```suggestion
HTTP redirects come as a new type of ACL configured within a frontend. You can fully customize the redirection URL, from the protocol scheme all the way to the query parameters, using powerful templated placeholders to reuse parts of the original requests.
```

> As well as this, we are also introducing the ability to create unconditioned ACLs, allowing you to explicitly define a default/catch all behavior for your frontend’s ACL.

Reviewer: ```suggestion
Furthermores, we are introducing the ability to create unconditioned ACLs, allowing you to explicitly define a default and catch all behavior for your frontend’s ACL.
```
